vyper/ast/parse.py

```diff
@@ -350,14 +342,16 @@ def visit_Expr(self, node):
         if isinstance(node.value, python_ast.Yield):
             # CMC 2024-03-03 consider unremoving this from the enclosing Expr
             node = node.value
-            node.ast_type = self._modification_offsets[(node.lineno, node.col_offset)]
+            node.ast_type = self._pre_parse_result.modification_offsets[
+                (node.lineno, node.col_offset)
+            ]
```

Comment:
let's try to avoid the multi-line formatting

```suggestion
            key = (node.lineno, node.col_offset)
            node.ast_type = self._pre_parse_result.modification_offsets[key]
```